### PR TITLE
value attribution DB interface Insert & Get method support

### PR DESCRIPTION
**Pull request by aligeti**
What: 
Support value attribution DB access methods
Why:
value attribution functionality
https://storjlabs.atlassian.net/browse/V3-1960
https://storjlabs.atlassian.net/browse/V3-1959
Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: pkg/valueattribution/valueattribution.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import (
+	"context"
+	"time"
+)
+
+// PartnerInfo describing connector/partner key info in the database
+type PartnerInfo struct {
+	PartnerID  []byte
+	BucketName []byte
+	CreatedAt  time.Time
```
Comment (coyle) [inline pkg/valueattribution/valueattribution.go]: I think we will just create this when inserting

Comment (aligeti) [reply pkg/valueattribution/valueattribution.go]: you mean pass these values as the insert()'s argument?

File: satellite/satellitedb/valueattribution.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+		BucketName: dbxInfo.BucketName,
+		CreatedAt:  dbxInfo.LastUpdated,
+	}, nil
+}
+
+// Insert implements create partner info
+func (keys *valueattributionDB) Insert(ctx context.Context, partnerinfo *valueattribution.PartnerInfo) (info *valueattribution.PartnerInfo, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	fmt.Println("KISHORE KISHORE --> partnerid=", dbx.ValueAttribution_PartnerId(partnerinfo.PartnerID))
```
Comment (coyle) [inline satellite/satellitedb/valueattribution.go]: prints

File: pkg/valueattribution/valueattribution.go [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+// DB implements the database for value attribution table
+type DB interface {
+	// GetByBucketName retrieves partner id using bucket name
+	GetByBucketName(ctx context.Context, buckname []byte) (*PartnerInfo, error)
```
[the remark this answers is not in the document]
Comment (aligeti) [reply pkg/valueattribution/valueattribution.go]: updated

File: satellite/satellitedb/valueattribution.go (hunk 2 above)
Comment (aligeti) [reply satellite/satellitedb/valueattribution.go]: oops.. cleanedup

File: pkg/valueattribution/valueattribution_test.go [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+			assert.NoError(t, err)
+
+			_, err = valAttrdb.Insert(ctx, partnerInfo2)
+			assert.Error(t, err)
+		}
+
+		{ // Get
+			info, err := valAttrdb.Get(ctx, partnerInfo.BucketName)
+			assert.NoError(t, err)
+			assert.Equal(t, partnerInfo.PartnerID, info.PartnerID)
```
Comment (mobyvb) [inline pkg/valueattribution/valueattribution_test.go]: also add get testcase for `partnerInfo1.BucketName`

**Pull request merged**